app/pages/resource/reservation-form/ReservationFormContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import ReservationForm from './ReservationForm';
+
+const resourceSelector = (state, props) => props.resource;
```

Comment:
Thanks for the thoughts guys, very good points!
Based on these ideas we had a quick discussion with Oskari and these are the guidelines we could go with:
- If there are any reusable selectors,  those should be defined in a separate place, for example `state/selectors` folder.
- In simple cases such as this one it's okay to have the selector for the container in the same file as the container itself.
- If the selector get's more complicated it should be moved to a separate file to improve readability.
- To avoid unnecessary work in moving selectors to separate file, we can have even simple selectors in their own file if we know that they are going to become more complicated in the future.
- The containers can have rendering functionality, especially in cases where there is no reusability benefit in separating the rendering to another Component. For example in cases like the page containers.
- For the `ReservationForm`and similar cases we can have a reusable component that can have multiple containers (in this case `ReservationCreateFormContainer` and `ReservationEditFormContainer`.

So go with what makes most sense and keep readability, reusability and testability in mind.